src/dashboard/src/components/administration/urls.py

```diff
@@ -29,6 +30,8 @@
     url(r'dips/atk/$', views.administration_atk_dips),
     url(r'dips/atom/$', views.atom_dips),
     url(r'dips/atom/edit_levels/$', views.atom_levels_of_description),
+    url(r'^i18n/', include('django.conf.urls.i18n')),
```

Comment:
What's the `i18n` url doing? I can't get anything at `administration/i18n/` and `administration/i18n/setlang/` just redirects back to transfers.